docs/api-reference/next/image.md

```diff
@@ -83,6 +85,10 @@ The layout behavior of the image as the viewport changes size.
   - When `fill`, the image will stretch both width and height to the dimensions of the parent element, provided the parent element is relative.
   - This is usually paired with the [`objectFit`](#objectFit) property.
   - Ensure the parent element has `position: relative` in their stylesheet.
+- When `raw`, the image will be rendered as a single image element with no wrappers, sizers or other responsive behavior.
+  - Unlike other layout modes, a `raw` image will pass through the `style` property to the underlying image.
```

Comment:
This is no longer true since it happens for all `layout` values

Reply:
Yep, removed.